### PR TITLE
feat: show Cloud Provider choice for Paid Plans on v1 billing

### PR DESCRIPTION
**Pull request by MildTomato**
## What kind of change does this PR introduce?

Bug fix, feature, docs update, ...

## What is the current behavior?

Please link any relevant issues here.

## What is the new behavior?

Feel free to include screenshots if it includes visual changes.

## Additional context

Add any other context or screenshots.

### CODE REVIEW

File: studio/pages/new/[slug].tsx [hunk 1]
```diff
@@ -436,39 +418,149 @@ const Wizard: NextPageWithLayout = () => {
...
-                          key={option}
-                          label={label}
-                          value={label}
-                          addOnBefore={() => (
-                            <img
-                              alt="region icon"
-                              className="w-5 rounded-sm"
-                              src={`${router.basePath}/img/regions/${
-                                Object.keys(availableRegions)[i]
-                              }.svg`}
```
Comment (alaister) [inline studio/pages/new/[slug].tsx:462]: Just making sure this doesn't get left out! We still need region selection ;)

**Review by alaister — COMMENTED** [review]
I hate to help Terry's cause, but the AWS logo in light mode isn't working so well:
Screenshot 2023-07-27 at 2 33 12 pm

File: studio/pages/new/[slug].tsx (hunk 1 above)
Comment (MildTomato) [reply studio/pages/new/[slug].tsx:462]: will fix

Comment (MildTomato) [reply studio/pages/new/[slug].tsx:462]: ha, how did i accidentally leave this out 😂 
I was reordering stuff to make more sense when 'free plan' was selected

**Comment by awalias** [comment]
closing as stale reopen as needed

**Pull request closed without merging**